src/integrationTest/groovy/wooga/gradle/unity/tasks/TestTaskIntegrationSpec.groovy

```diff
@@ -282,4 +412,22 @@ class TestTaskIntegrationSpec extends UnityTaskIntegrationSpec<Test> {
 
     }
 
+    File createAppConfig(String appConfigFullFileName, String batchModeBuildTarget) {
+        def appConfigFile = createFile(appConfigFullFileName, projectDir)
+        def appConfig = [
+                'MonoBehaviour': ['bundleId': 'net.wooga.test', 'batchModeBuildTarget': batchModeBuildTarget]
+        ]
+        def UNITY_ASSET_HEADER = """
+            %YAML 1.1
+            %TAG !u! tag:unity3d.com,2011:
+            --- !u!114 &11400000
+            """.stripIndent().trim()
+        appConfigFile << UNITY_ASSET_HEADER
+        appConfigFile << "\n"
+        Yaml yaml = new Yaml()
+        appConfigFile << yaml.dump(appConfig)
+
+        return appConfigFile
+    }
```

Comment:
This should not be in this repo. For a testsetup in this project no appconfigs are needed.

Reply:
Done